string node_id
string mqtt_topic
---
bool subscription_success
